kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ProcessServiceIntegrationTest.java

```diff
@@ -178,6 +189,28 @@ public void testAbortExistingProcess() {
             fail(e.getMessage());
         }
     }
+    
+    @Test()
+    public void testAbortExistingProcessWrongContainer() {
+        Map<String, Object> parameters = new HashMap<>();
+
+        Long processInstanceId = processClient.startProcess(CONTAINER_ID, PROCESS_ID_EVALUATION, parameters);
+        try {
+            assertNotNull(processInstanceId);
+            assertTrue(processInstanceId > 0);
+
+            // Process instance is running and is active.
+            ProcessInstance processInstance = processClient.getProcessInstance(CONTAINER_ID, processInstanceId);
+            assertNotNull(processInstance);
+            assertEquals(org.kie.api.runtime.process.ProcessInstance.STATE_ACTIVE, processInstance.getState().intValue());
+            exceptionRule.expect(KieServicesException.class);
```

Comment:
Why don't add also
`exceptionRule.expectMessage(FUNNY_CONTAINER_ID);`

Reply:
hmmm, not really, if the container exist, but it is different from the one that the process intance was created into, the exception being launched will not necessarily contain the contaner id. The engine will locate a session, but that session will not include the id.
Since this is implementation detail, I thik I will remove expect message from the test in what it was used. 
I will alsotry to  test the scenario in which the container exist but does not hold that processInstance